declarations_site/declarations_site/settings.py

```diff
@@ -241,7 +241,7 @@
     },
 }
 
-SITEMAP_DECLARATIONS_PER_PAGE = 50000
+SITEMAP_DECLARATIONS_PER_PAGE = 10000
```

Comment:
Can you update sitemap code then to perform 5 requests and combine the results then? I'm afraid a bit, that radical change in sitemaps can make google sick a bit.